tests/apps/dial/dial_progress/test-config.yaml

```diff
@@ -42,7 +42,7 @@ ami-config:
             requirements:
                 match:
                     Result: 'Pass'
-            count: 2
+            count: 3
```

Comment:
First, get rid of the hangup monitor stuff.  It's not helping.

Second, setting `count: 3` doesn't actually cause the test case to wait for 3 occurrences.  You want to do something like...
```
ami-config:
    -
        ami-events:
            conditions:
                match:
                    Event: 'UserEvent'
                    UserEvent: 'DialProgress'
            requirements:
                match:
                    Result: 'Pass'
            count: 3
    -
        ami-events:
            conditions:
                match:
                    Event: 'Hangup'
                    Channel: 'Local/s@default-00000000;1'
            count: 1
        stop_test:
```

Now the test will stop on the final Hangup and will then check that there were 3 occurrences of the UserEvent.

When I fixed the test up locally, it took 10 seconds and passed.